manage-data/migrate/migrate-with-logstash.md

```diff
@@ -0,0 +1,119 @@
+---
+navigation_title: Migrate Elastic Cloud Hosted data to Serverless with Logstash
+applies_to:
+  stack: ga
+  deployment:
+    ess: ga
+    ece: ga
+products:
+  - id: elasticsearch
+  - id: logstash
+  - id: cloud-hosted
+  - id: cloud-enterprise
+  - id: cloud-kubernetes
```

Comment:
Same here. The steps might work for ECE and ECK deployments, but that's not mentioned on the page, so perhaps these two product types shouldn't be included here.